_includes/ontology_table.html

```diff
@@ -77,7 +77,7 @@
       <td>
         {% if ont.contact %}
         <a href="mailto:{{ont.contact.email}}">
-          <button type="button" class="btn btn-default" aria-label="Left Align" title="Email ontology devs">
+          <button type="button" class="btn btn-default" aria-label="Send an email to {{ont.contact.label}}, the contact person for {{ ont.title }}" title="Email ontology devs">
```

Comment:
What happens if there is no contact? Pages are still rendered for obsolete ontologies?

Reply:
There’s an if statement above that checks this

Reply:
oops. thx